`/home/tastyfish/Git/OpenMF/components/6ds/parser.cpp:11:26: error: ISO C++ forbids comparison between pointer and integer [-fpermissive]`

Should use strcmp or something.

--- a/components/6ds/parser.hpp
+++ b/components/6ds/parser.hpp
@@ -55,7 +55,7 @@ bool DataFormat6DS::load(std::ifstream &srcFile)
     Header header = {};
     read(srcFile, &header);
 
-    if (header.mMagic != '6DS')
+    if (std::string(header.mMagic) != "6DS")
     {
         return false;
     }

